src/features/MovieCategories/component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import './style.scss';
+
+export default class MovieCategories extends Component 
+{
+    // define our state
+    constructor(props)
+{
+        super(props);
+    }
+
+    render() 
+{
+        return (
+	<aside className='movieSection-categories'>
+		<ul className='category-items'>
+			{this.props.categories.map((item, index) => (
+				<h4 
+					key={index} 
+					className={item.active ? 'category-item active' : 'category-item'}
```

Comment:
Add package https://www.npmjs.com/package/clsx
it is very lightweight though gives some flexibility when working with classes